src/include/stir/KOSMAPOSL/KOSMAPOSLReconstruction.h

```diff
@@ -208,12 +211,14 @@ class KOSMAPOSLReconstruction:
   shared_ptr<TargetT> kpnorm_sptr;
  //kernel parameters
   int num_neighbours,num_non_zero_feat,num_elem_neighbourhood,num_voxels,dimz,dimy,dimx;
+  int stop_iterative_kernel_at_iter_num;
   std::vector<double> sigma_m;
   bool only_2D;
   bool hybrid;
   double sigma_p;
   double sigma_dp, sigma_dm;
   BasicCoordinate<3,int> min_ind, max_ind;
+  shared_ptr<TargetT> iterative_kernel_image_fixed_sptr;
```

Comment:
Possibly could rename/simplify to `iterative_kernel_image_sptr`? It may or may not be fixed, just represents a sptr to the image for the iterative kernel.

Reply:
Mh actually this is meant to be fixed to whatever we chose in the parfile